cmd/runtimecfg/render.go

```diff
@@ -43,10 +46,24 @@ func runRender(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		apiVip = nil
 	}
+	apiVips, err := cmd.Flags().GetIPSlice("api-vips")
+	if err != nil {
+		apiVips = []net.IP{}
+	}
```

Comment:
Sure, I can do that. Worth noting that all of this backward compat code will break if/when we remove the deprecated params, so there's not really a concern about forgetting it.

Reply:
Since I missed doing this in the latest patch set here, I'm going to push a followup to handle the comment change so we don't have to wait for ci on this again. The dual stack vip work is now blocking other people because the api revendoring in MCO needs this.